### ISSUE TITLE
no .lexc file

### ISSUE THREAD

**Issue by jonorthwash** [apertium/apertium-sme-nob#1]
`README` says the following exists, but it doesn't seem to:
`* apertium-sme-nob.sme.lexc            - Monolingual dictionary for Sámi`

Either the documentation needs to be fixed, or the file should be added.

**Comment by Trondtr** [comment]
Yes, this is outdated by many years. We (and you, if you want to run this yourself) now compile sme in the giella infrastructure, and point to it from the apertium catalogue. The setup is documented here:
http://wiki.apertium.org/wiki/Northern_Sámi_and_Norwegian/Installation 
This text may be a bit minimalistic, this one says more:
http://giellatekno.uit.no/doc/infra/GettingStarted.html

What is referred to as a "monolingual dictionary for Sámi" is the (numerous) set of files in this folder:
https://victorio.uit.no/langtech/trunk/langs/sme/src/morphology/
(all files with an .lexc suffix in this folder and its subfolders). 

You may inspect the files from this link, but if you want to compile the fsts, you should follow the documentation. And yes, the documentation should be fixed.

**Comment by Trondtr** [comment]
There I changed the two obsolete lines, have a look.

**Comment by jonorthwash** [comment]
Better, but I guess that makes these parts problematic too:

> Given the requirements being installed, you should be able to just run:

and

> * apertium-sme-nob.nob.dix             - Monolingual dictionary for Bokmål
